src/pages/home/Home.tsx

```diff
@@ -36,13 +39,17 @@ export default function Home() {
           </div>
         ) : (
           <>
-            {data && (
+            {data ? (
               <h2 className="text-3xl mt-8">
-                Showing {data?.results.length} of {data?.count} results
+                Showing {data.results.length} of {data.count} results
               </h2>
-            )}
+            ) : null}
             <div className="py-16 grid gap-8 xs:gap-16 grid-cols-1 xs:grid-cols-2 md:grid-cols-3 lg:grid-cols-4 xl:grid-cols-5 justify-items-center">
-              {data && data.results.map((result) => <Book key={result.id} book={{ ...result }} />)}
+              {data
+                ? data.results.map((result) => (
+                    <BookCard key={result.id} book={{ ...result }} />
+                  ))
```

Comment:
nie musisz tego spreadować, wystarczy przekazać obiekt
```suggestion
                ? data.results.map((result) => (
                    <BookCard key={result.id} book={result} />
                  ))
```